channels/by.m3u

```diff
@@ -1,25 +1,25 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="8kanalVitebsk.by" tvg-name="8 канал (Витебск)" tvg-country="BY" tvg-language="" tvg-logo="" group-title="",8 канал (Витебск)
+#EXTINF:-1 tvg-id="8kanal.by" tvg-name="8 канал" tvg-country="BY" tvg-language="" tvg-logo="" group-title="",8 канал (Витебск) (576p)
```

Comment:
Doesn't seem to be 24/7

![vlcsnap-2021-08-14-00h45m30s944](https://user-images.githubusercontent.com/30985701/129424872-aa8488e3-09f9-4a2b-9b1d-4f846ee8ec9c.png)